app/controllers/hosts_controller.rb

```diff
@@ -135,6 +135,15 @@ def compute_resource_selected
 
   def interfaces
     @host = Host.new params[:host]
+
+    if @host.compute_profile_id && @host.compute_resource_id
+      profile = @host.compute_resource.compute_attributes.find_by_compute_profile_id(@host.compute_profile_id)
+      if profile
+        merge = InterfaceMerge.new
```

Comment:
just and idea... would it make more sense to pass just @host so InterfaceMerge would do the merging and returned host with merged interfaces? this `@host.interfaces` and compute_profile logic should not live in controller I think